### PR TITLE
docs: user setup

### PR DESCRIPTION
**Pull request by klesh**
# Summary

User setup documentation

### Key Points

- [x] This is a breaking change
- [x] New or existing documentation is updated

### Description
User can set up devlake with docker-compose

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ run:
 	go run main.go
 
 compose:
-	docker-compose up
+	docker-compose up grafana
```
Comment (likyh) [inline Makefile:18]: what's this?

Comment (klesh) [reply Makefile:18]: for developer, this launch granfan and mysql,  without lake

**Comment by likyh** [comment]
LGTM

**Review by likyh — APPROVED** [review]
LGTM

**Pull request merged**